frontend/src/routes/explore/+page.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+    import data from "$lib/data";
+</script>
+
+<!-- this fixes the issue of weird extra space to the right of the page -->
+<div
+    class="columns my-0"
+    style="width: calc(100vw + 12px); height: calc(100vh - 56px);"
+>
+    <div class="column is-one-quarter has-background-grey-darker">
+        <nav class="level mx-4">
+            <div class="level-left">
+                <div class="level-item">
+                    <h1 class="title is-5">Watchlist</h1>
+                </div>
+            </div>
+
+            <div class="level-right">
+                <div class="level-item">
+                    <span class="icon">
+                        <a href="/"><i class="fas fa-plus" /></a>
+                    </span>
+                </div>
+            </div>
+        </nav>
+        <slot />
+    </div>
+
+    <div class="column">
+        <nav class="level mx-2">
+            <div class="level-left">
+                <div class="level-item">
+                    <h1 class="subtitle is-5">Apple Inc.</h1>
+                </div>
+                <div class="level-item">
+                    <h1 class="subtitle is-5">NASDAQ</h1>
+                </div>
+            </div>
+
+            <div class="level-right">
+                <div class="level-item">
+                    <span class="icon">
+                        <a href="/"><i class="fas fa-magnifying-glass" /></a>
+                    </span>
+                </div>
+            </div>
+        </nav>
+        <div
+            class="block mx-2"
+            style="height: 50vh; display: flex; justify-content: center; align-items: center;"
+        >
+            Graph
+        </div>
+
+        <hr style="background: #4a4a4a;" />
```

Comment:
Could replace this with:
```
<style lang="scss">
    hr {
        background: $grey-dark;
    }
</style>
```

VSCode will give warning when opened in Workspace, to get rid of it you can open the frontend folder in its own VSCode window. Feel free to ignore this, depends which you find more annoying :)